.github/workflows/ci.yml

```diff
@@ -0,0 +1,51 @@
+name: Ruby-ci
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+env:
+  ACCOUNT_ID: ${{ secrets.ACCOUNT_ID }}
+  API_KEY: ${{ secrets.API_KEY }}
+
+permissions:
+  contents: read
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ruby-version: ['2.4.2']
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@55283cc23133118229fd3f97f9336ee23a179fcf # v1.146.0
+        with:
+          ruby-version: ${{ matrix.ruby-version }}
+          bundler-cache: true # runs 'bundle install' and caches installed gems automatically
+      - name: Run tests
+        run: bundle exec rspec --format RspecJunitFormatter --out test_results/rspec.xml --format progress
```

Comment:
This also needs to run 
```
bundle check
bundle install
```

Reply:
@jlin-sift Since we are doing `bundler-cache: true # runs 'bundle install' and caches installed gems automatically` in line #30 per my understanding we don't need to execute the `check || install` specifically.

Reply:
OK. In that case, do you want to remove `bundle check` and `bundle install` from `run_integration_tests`?

Reply:
removed